filenames.md

```diff
@@ -1,7 +1,249 @@
+not just in filenames). ISO8601 is an international standard for the representation
+of dates and times.
+
+For filenames though, you **MUST** use the _basic_ format of ISO8601, which
+does not include any separators (e.g. hyphens or colons). This produces a
+more compact datestamp and avoids any issues with special characters (mainly `:`)
+in filenames.
+
+Filenames **SHOULD** contain a UTC offset (see below) to avoid ambiguity about the
+time the recording started. Filenames **MAY** use `Z` to indicate UTC+0.
+
+Examples of the recommended format follow:
+
+```txt
+20170625T102105Z.flac
+20250990T033235.123456+0930.wav
+20250990T033235-0430.wav
+```
+
+Variations of this format **MAY** omit the UTC offset, e.g. `20170625T102105.flac`
+but this is **NOT RECOMMENDED** because the datestamp becomes ambiguous.
+
+### Precise format
+
+The precise format for the date and time is: `YYYYMMDDThhmmss[.SSSSSS][Z|(+|-)hhmm]`.
+Where:
+
+| Code       | Meaning                                                             |
+| ---------- | ------------------------------------------------------------------- |
+| YYYY       | A 4 digit year                                                      |
+| MM         | A 2 digit month, with leading zeros                                 |
+| DD         | A 2 digit day, with leading zeros                                   |
+| T          | A separator between the data and time components                    |
+| hh         | Hours, 0-24, with leading zeros                                     |
+| mm         | Minutes, with leading zeros                                         |
+| ss         | Seconds, with leading zeros                                         |
+| SSSSSS     | [Optional] Fractional seconds (up to 6 digits)                      |
+| Z          | [Optional] UTC+0 designator                                         |
+| (+\|-)hhmm | [Optional] UTC offset in hours and minutes, positive if east of UTC |
+
+## Locations
+
+It is **recommended** to encode location information inside the metadata of
+audio files rather than in filenames.
+
+However, if you do need to encode location, then filenames **MUST** use
+[ISO6709:H](<https://en.wikipedia.org/wiki/ISO_6709#String_expression_(Annex_H)>)
+without a trailing solidus to represent locations in filenames.
+We omit the trailing solidus (`/`) to avoid special characters in filenames.
+
+The precise format is: `(+|-)<Latitude>(+|-)<Longitude>[[+|-]<Altitude>][<CRS>]`
+, where:
+
+| Code      | Meaning                                                   |
+| --------- | --------------------------------------------------------- |
+| Latitude  | Latitude in decimal degrees, DD.DDDDDD                    |
+| Longitude | Longitude in decimal degrees, DDD.DDDDDD                  |
+| Altitude  | Altitude in meters above sea level (optional)             |
+| CRS       | Coordinate Reference System (optional, default is WGS84). |
```

Comment:
Above you used an `[OPTIONAL]` prefix, but here you use `(optional)` suffix?

<img width="683" height="86" alt="Image" src="https://github.com/user-attachments/assets/5a70b381-9ec6-486f-b47c-68360112fb41" />